docs/resources/google_sourcerepo_repositories.md

```diff
@@ -0,0 +1,31 @@
+---
+title: About the google_sourcerepo_repositories resource
+platform: gcp
+---
+
+## Syntax
+A `google_sourcerepo_repositories` is used to test a Google Repository resource
+
+## Examples
+```
+repo_name = 'inspec-gcp-repository'
+describe.one do
+  google_sourcerepo_repositories(project: 'chef-gcp-inspec').names.each do |name|
+    describe name do
+      it { should match /\/repos\/#{repo_name}$/ }
+    end
+  end
+end
+```
+
+## Properties
+Properties that can be accessed from the `google_sourcerepo_repositories` resource:
+
+See [google_sourcerepo_repository.md](google_sourcerepo_repository.md) for more detailed information
+  * `names`: an array of `google_sourcerepo_repository` name
+  * `urls`: an array of `google_sourcerepo_repository` url
+  * `sizes`: an array of `google_sourcerepo_repository` size
+
+## Filter Criteria
+This resource supports all of the above properties as filter criteria, which can be used
+with `where` as a block or a method.
```

Comment:
Could we add something to the documentation here to explain that the Cloud Source Repository API should be enabled?  Might be worthwhile doing the same for other new services e.g. 
explaining that the Cloud Source Repository API needs to be enabled.  Here's a similar example for compute engine:

``` 
## GCP Permissions

Ensure the [Compute Engine API](https://console.cloud.google.com/apis/library/compute.googleapis.com/) is enabled for the current project.
```

Reply:
Good idea! Added this on all generated docs

Reply:
Awesome, thanks @slevenick !